### PR TITLE
Feature, add support for light platform

### PR DESCRIPTION
**Pull request by fredrike**
I'm not sure how HALO would react to this..

### CODE REVIEW

File: custom_components/chargeamps/light.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        return status not in (False, "Off")
+
+    async def async_turn_on(self, brightness=None):
+        if brightness:
+            if brightness < 128:
+                brightness = "low"
+            elif brightness < 192:
+                brightness = "medium"
+            else:
+                brightness = "high"
```
Comment (fredrike) [inline custom_components/chargeamps/light.py:66]: I do like this, but it is not really pythonic..


```suggestion
            brightness = {brightness<128: 'low', 128<=brightness<192: 'medium'}.get(True, 'high')
```

Comment (jschlyter) [reply custom_components/chargeamps/light.py:66]: That last line is wicked! Please do if/elif/else for readability, but F for Fun on the shorter version :-)

**Comment by jschlyter** [comment]
Merged manually.

**Comment by fredrike** [comment]
@jschlyter does this work on Aura? I read trough the api description and it seems legit.

Also, when can I expect a new release?

**Comment by jschlyter** [comment]
Both our Halo and Auro is down for maintenance at the moment, @fredriklj will test as soon as they are back online.

**Pull request closed without merging**